conduct.md

```diff
@@ -7,16 +7,282 @@
+is obligated to adhere to the 9 categories in the Code of Conduct. They will be
+our foundation throughout the semester when working together. Moving on, everyone
+should respect one another; no one should be excluded based on their race or gender.
+Team members should not shun the ideas of anyone, especially those who may
+have not have had an equal amount of experience as others. Also, at the beginning
+of each session, everyone should enter with positive attitudes, These positive
+attitudes should stay consistent whether the project is working or breaking.
+
+* **Acceptable Conduct (A):**
+  * Respect to all peers
+  * Allowing all members to voice their opinion
+  * Showing positive attitudes towards all aspects of the project
+
+* **Unacceptable Conduct (U):**
+  * Excluding members of team based on their race or gender
+  * Refusal to follow the rest of the categories in the Code of Conduct
+  * Showing a negative attitude towards group members or aspects of the project
+
+## Deadlines
+
+To determine appropriate deadlines conduct policies it is first important to
+break up deadlines into the different types of possible deadlines for the
+course. The deadlines categories that are possible are those when working
+directly with group members to finish work necessary to move onto other parts
+of the lab, along with completing your assigned portion of the work by the due
+date of the lab.
+
+* **Acceptable Behavior (C)**
+
+* Talk with team members and discuss work needed to be done.
+
+* Set up a reasonable time frame for when the work should be completed
+  leaving time for the rest of the lab.
+
+* Take into consideration other people's schedules and activities as it may
+  not align with yours.
+
+* If you can not come up with an agreed upon time frame, have a vote
+  between teammates that are present to determine the time frame.
+
+* If problems still arise then seek out other team members.
+
+* Grade yourself based on the grading scale given to determine how well you
+  followed the deadlines section. Grading scale is as follows,
+
+* All software project assignments completed have a specified due date, whereas
+  the electronic copy must be uploaded to GitHub through the designated repository
+  before the assigned deadline.
+
+* Software projects will be completed according to the deadline set by
+  the students, and within the one week period before the next laboratory session
+  begins.
+
+* **Unacceptable Behavior (U)**
+
+* Not discussing things with team members on due dates.
+
+* Not completing work on time.
+
+* Not participating in votes on material.
+
+* Not communicating reasons for work being late and not on time.
+
+* If work is submitted after the specified deadline, the assignment will not be
+  accepted and no credit will be given (unless specified otherwise). All submitted
+  software project assignments must contain high-quality, well documented code.
+
+## Communication
```

Comment:
The indentation of the bullets in this section should be changed to look like the ones in the conflicts section since both sections are using the format of members/team leads shall...

Reply:
Fixed in a previous PR.